src/aggregate_utils.py

```diff
@@ -24,22 +46,45 @@ def download_all_promos(verbose=True):
     all_promos = []
 
     for cursor in range(0, total_num_promos, step):
-        store_data = download_store_data(cursor=cursor, step=step, verbose=verbose)
+        store_data = download_store_data(
+            cursor=cursor, step=step, verbose=verbose)
```

Comment:
I would run [`black` formatter](https://github.com/psf/black) before any merge.

Reply:
I was using `autopep8`, sorry, I should've checked!